docs/leverage.md

```diff
@@ -1,21 +1,80 @@
 # Leverage
+*You can't run 2 bots on the same account with leverage*
+
+!!! Warning "Trading with leverage is very risky"
+    *(`trading_mode="margin"` or `trading_mode="futures"`)* Do not trade with a leverage > 1 using a strategy that hasn't shown positive results in a live run using the spot market. Check the stoploss of your strategy. With a leverage of 2, a stoploss of 0.5 would be too low, and these trades would be liquidated before reaching that amount
+
+> I've only been using freqtrade for a couple weeks, but I feel like I'm pretty good and could use leverage
+
+**No you're not. Do not use leverage yet.**
+### Understand trading_mode
+
+The possible values are: `spot` (default), `margin`(*coming soon*) or `futures`.
+
+**SPOT**
+    Regular trading mode. 
+        - Shorting is not available
+        - There is no liquidation price
+        - Profits gained/lost are equal to the change in value of the assets(minus trading fees)
+
+#### Leverage trading modes
+
+# TODO: include a resource to help calculate stoplosses that are above the liquidation price
+
+#TODO: Taken from investopedia, is that ok?
```

Comment:
Should be - maybe put a "source" link to the bottom of it - so it's clear where it's been sourced (and i assume Investopedia has more info on this - so people can go read more there)...